packages/autorest.python/autorest/codegen/models/client.py

```diff
@@ -325,7 +326,7 @@ def sdk_moniker(self) -> str:
     def name(self) -> str:
         return f"{super().name}Configuration"
 
-    def imports(self, async_mode: bool) -> FileImport:
+    def _imports_shared(self, async_mode: bool) -> FileImport:
```

Comment:
i really don't think we need the `for_multiapi`, `import_literal` arguments, it makes the code a lot less clean. I started working on getting rid of it, it's getting close but I wasn't able to finish today. I'm going to push waht I've been working on into a commit "initial code clean up", can you try following it and making it work? Thanks!

Reply:
besides that i think it's almost ready to merge, just really don't want to take on that technical debt bc idt we need to

Reply:
Hi @iscai-msft ,  
I added `imports_for_multiapi` on type to make generated code clean.

And now the remaining issue in pipeline is that the mypy complains for Literal[float/others], and pylance also report same error in vscode:
![image](https://user-images.githubusercontent.com/59815250/196893891-966084d8-369c-4d63-9534-960c90540b5e.png)
I imagine two options for these errors:
1. not to use Literal on these types,
2. add `# type: ignore` behind these Literals

for the second option, the intellisense still show red line on the `Literal`, and I'm wondering it's also a little confused that we declared a type but ask mypy to ignore it.

Reply:
I agree, we should do #1, sorry for removing that code earlier!

Reply:
done for #1